config/locales/devise.pt.yml

```diff
@@ -1,6 +1,6 @@
 # Additional translations at https://github.com/plataformatec/devise/wiki/I18n
 
-en:
+pt:
```

Comment:
Change it to `en` and change the default locale. The site's in english either way.